spec/models/user_spec.rb

```diff
@@ -25,6 +25,12 @@
 
   it { is_expected.to be_valid }
 
+  context "when user has email address with apostrophe" do
+    subject(:user) { create(:user, email: "o'niel@example.com") }
```

Comment:
Agreed on `build`. I prefer `build` for validation tests anyway, since if validation fails, the `create` would blow up before hitting the actual test.